projects/js-packages/components/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ### This is a list detailing changes for the Jetpack RNA Components package releases.
 
+## 0.10.3 - 2022-01-25
+### Changed
+- Do not style header elements from AdminSection component
```

Comment:
I'm assuming we don't care for projects like this to ensure all entries have a trailing period? The plugins are the critical ones to look good?

Reply:
Right, exactly.